### DISCUSSION TITLE
multilingual?

### DISCUSSION THREAD

**Discussion by daslicht** [apostrophecms/apostrophe#3220] [Q&A] [answered]
Is it possible to create multilingual webpages with Apostrophe 3 ?
Complete translated webpages for _anything_.

**Comment by boutell** [comment, marked as answer]
Hi Marc,

This is our priority in the current set of development sprints. We'll be
rolling out static localization (i.e. for text in templates) followed by
dynamic localization (for your page content) in the next few weeks.

On Sun, Jul 4, 2021 at 4:07 AM Marc Wensauer ***@***.***>
wrote:

> Is it possible to create multilingual webpages with Apostrophe 3 ?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apostrophecms/apostrophe/discussions/3220>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAH27JDKGO2CGQB7DDCGULTWAJEFANCNFSM47Y6I5VA>
> .
>


-- 

THOMAS BOUTELL | CHIEF TECHNOLOGY OFFICER
APOSTROPHECMS | apostrophecms.com | he/him/his

**Comment by abea** [comment]
See today's release notes for more on the progress of this feature set: https://github.com/apostrophecms/apostrophe/discussions/3236
